changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "62535136-e63d-4727-ae28-84d7a9ed964a",
+          "ApiChange": "Resource",
+          "ChangedApiName": "crossTenantAccessPolicyConfigurationBase",
+          "ChangeType": "Removal",
+          "Description": "Removed the crossTenantAccessPolicyConfigurationBase resource.",
```

Should be `Deleted`.
```suggestion
          "ChangeType": "Deletion",
          "Description": "Deleted the **crossTenantAccessPolicyConfigurationBase** resource.",
```